### ISSUE TITLE
refactor and create classes

### ISSUE THREAD

**Issue by oimartin** [dictybase-playground/learn-github-action#21]
email.py --> convert to class
connect.py+markdwn2html.py --> convert to class

**Comment by oimartin** [comment]
update:
GithubIssue class
   - need to use named tuple to pass information to the class (org, repo, token)
   - don't pass the commandline
   - (org, repo, token) = state
   - get rid of github_repo() method
   - methods: 
                     - given issue id return issue
                     - given issue return body
                     - given body return html
 EmailUser class
   - api & key reused =  state
   - method:  - send email

**Comment by oimartin** [comment]
For GithubIssue, I created a new module, githubrepo.py to create a named tuple with the info to connect to Github. Then the other two methods within the class are get_body and mrkdwn_html. 

EmailUser: 
I redid dict.py to be a namedtuple and renamed it orderinfo.py I just used that one namedtuple to be called by the class and by the method, send_email. I could also create two separate named tuples within orderinfo.py to call in separately.
